html/index.html

```diff
@@ -0,0 +1,332 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8" />
+    <title>What is my IP address? &mdash; {{ .Host }}</title>
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <meta
+      name="description"
+      content="ifconfig. What is my IP address? &mdash; The best tool to find your own IP address, and information about it."
+    />
+    <link rel="canonical" href="https://ifconfig.co/" />
```

Comment:
Should be replaced with `{{ .Host }}`, for selfhosted use-case?

Reply:
This should point to the one only ifconfig.co. See https://developers.google.com/search/docs/advanced/crawling/consolidate-duplicate-urls

Otherwise, if people use a google search google may have many pages which look exactly the same, but hosted on different domains. It may then decide to distribute the search ranking over all of them. -- making ifconfig.co less findable. 

This tag tells search engines that ifconfig.co is the "canonical" URL